components/Card/index.js

```diff
@@ -26,11 +29,26 @@ const StyledCountry = styled.li`
   border: 2px solid black;
   width: 90%;
   text-align: left;
-  background-color: turquoise;
+  background-color: var(--primary-color);
   border-radius: 6px;
   padding-top: 8%;
   padding-bottom: 8%;
   padding-left: 2%;
   font-size: x-large;
   margin-top: 0 auto;
 `;
+
+const StyledDiv = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-bottom: 5%;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  border: 4px solid black;
+  border-radius: 50%;
+  padding: 0px 12px 4px 12px;
+  color: black;
+  font-size: xx-large;
```

Comment:
I personally prefer number-based values rather than opaque values like "big", "not so big", "probably small, depending on the screen size" 😂 . I know this is built-in, I just don't like it.